src/main/java/basestation/bot/commands/CommandCenter.java

```diff
@@ -8,8 +8,45 @@
  * If there are any important commands we believe all robots should have then place them here
  */
 public interface CommandCenter {
+	/**
+     * Starts logging of data
+     */
+    void startLogging();
+
+    /**
+     * Returns whether or not data is currently being logged.
+     * @return True if the command center is currently logging data
+     */
+    boolean isLogging();
+
+    /**
+     * Stores key value pair. 
```

Comment:
also it looks like this doesn't account for instantaneous vs not instantaneous commands? I thought there would be a third boolean to represent that.